frontend/messages/fr.json

```diff
@@ -367,7 +367,7 @@
 	"extraLarge": "Très grand",
 	"rid": "RID",
 	"scope": "Périmètre",
-	"auditor": "Auditeur",
+	"reader": "Auditeur",
```

Comment:
The translation in french stays the same @eric-intuitem?

Reply:
Indeed. Solved.